.circleci/config.yml

```diff
@@ -149,7 +149,7 @@ jobs:
 
       - run:
           name: "clone go-ethereum from infinitism"
-          command: git clone -b RIP-7560-revision-2 --single-branch https://github.com/eth-infinitism/go-ethereum.git
+          command: git clone -b AA-381-create-getRip7560Bundle-api --single-branch https://github.com/eth-infinitism/go-ethereum.git
```

Comment:
avoid naming branches directly, as it can't be merged correctly later.
instead, use same-branch-name (after PR #94 is included)

Reply:
No longer relevant - merged the script helper from master